### DISCUSSION TITLE
Laravel - API and Cloudfront

### DISCUSSION THREAD

**Discussion by craigwillis85** [brefphp/bref#1247] [Support]
Hi all

I have a basic Laravel App that is just an API. There is no assets as such (CSS or JS), just a few basic HTTP requests.

I have successfully deployed to AWS and everything is working fine when accessing the API gateway.

I have my own domain, which is not managed in AWS and I tried to create a CNAME using the API Gateway URL, but got a "Forbidden" (I think this is expected).

Am I correct in thinking I need to map the API Gateway URL to Cloudfront? If so, can someone point me to an example as I've not found any example in the Bref docs

Thanks

**Comment by mnapoli** [comment]
Hi, here is the documentation you need: https://bref.sh/docs/environment/custom-domains.html

**Comment by felipe-dsdev** [comment]
@craigwillis85 This is the plugin you need: https://www.serverless.com/plugins/serverless-domain-manager
